bulk_query_gen/influxdb/influx_common.go

```diff
@@ -40,7 +42,7 @@ func (d *InfluxCommon) getHttpQuery(humanLabel, intervalStart, query string, q *
 	q.HumanDescription = []byte(fmt.Sprintf("%s: %s", humanLabel, intervalStart))
 	q.Language = d.language.String()
 
-	if d.language == InfluxQL {
+	if d.language == InfluxQL && d.version == 1 {
```

Comment:
Looking at this again after seeing the full error messages, I don't think this change is what we want. It looks like V2 is unconditionally sending queries to `/api/v2/query`, which is coded to only accept POST queries. InfluxQL queries on V2 must be sent to `/query` instead. There's a whole auth mechanism for InfluxQL-on-v2 that's separate from Flux-on-v2's token auth... we could probably say that `-token` on V2 + InfluxQL should be a `username:password` pair for use in basic auth.

Reply:
(I can run with trying to implement the above, just wanted to put it in writing so I don't forget)

Reply:
In https://github.com/influxdata/influxdb-comparisons/pull/186, I made an update to the query benchmarker that would allow for running InfluxQL queries against a 2.x instance using the `/query` endpoint using [token authentication](https://docs.influxdata.com/influxdb/v2.0/reference/api/influxdb-1x/#authenticate-with-the-token-scheme). With a little bit of finagling that should work with basic auth too, although for benchmarking purposes I think we could just use token authentication since we'll already have the token available. I'm not totally up to speed on how the v1 auth works though so I might be missing something.